torchaudio/datasets/gtzan.py

```diff
@@ -0,0 +1,1064 @@
+import os
+from typing import Any, Tuple
+
+import torchaudio
+from torch import Tensor
+from torch.utils.data import Dataset
+from torchaudio.datasets.utils import (
+    download_url,
+    extract_archive,
+    walk_files,
+)
+
+# The following lists prefixed with `filtered_` provide a filtered split
+# that:
+#
+# a. Mitigate a known issue with GTZAN (duplication)
+#
+# b. Provide a standard split for testing it against other
+#    methods (e.g. the one in jordipons/sklearn-audio-transfer-learning).
+#
+# Those are used when GTZAN is initialised with the `filtered` keyword.
+# The split was taken from (github) jordipons/sklearn-audio-transfer-learning.
```

Comment:
Are there other splits that people use instead?

Reply:
AFAIK yes but this one has been used in a previous ISMIR late breaking demo by Jordi Pons and Xavier Serra:

```Jordi Pons, Xavier Serra.  “musicnn:  pre-trained convolutional neural networks for music audio tagging”, Late Break-ing/Demo at the 20th International Society for Music Information Retrieval, Delft, The Netherlands, 2019.```

As well as their transfer learning with MusiCNN:
https://github.com/jordipons/sklearn-audio-transfer-learning/